internal/resolution/variablesources/bundle.go

```diff
@@ -1,63 +1,34 @@
 package variablesources
```

Comment:
I initially had everything in this package in two files: `olm.go` and `olm_test.go`. While `olm.go` was is still manageable, navigating `olm_test.go` was a bit difficult. So I decided to split into multiple files.

It resulted in a more complicated diff (while the files themselves are straightforward).